crates/neon/src/context/mod.rs

```diff
@@ -203,6 +206,34 @@ impl CallbackInfo<'_> {
         unsafe { sys::call::argv(cx.env().to_raw(), self.info) }
     }
 
+    #[cfg(feature = "serde")]
+    pub(crate) fn argv_exact<'b, C: Context<'b>, const N: usize>(
+        &self,
+        cx: &mut C,
+    ) -> [Handle<'b, JsValue>; N] {
+        use std::ptr;
+
+        let mut argv = [JsValue::new_internal(ptr::null_mut()); N];
```

Comment:
I think it's worth adding a comment here that Node-API fills slots with undefined if the number of actual arguments is smaller than the length of argv.

Reply:
I should probably also document that the safety here depends on `Handle<JsValue>` being a transparent wrapper for a pointer.